plugins/workbench-resources/src/components/AccountPopup.svelte

```diff
@@ -45,29 +43,31 @@
   }, { limit: 1 })
 
 
-  function selectSpecial (sp: SpecialNavModel): void {
+  function selectCategory (sp: SettingsCategory): void {
+    closePopup()
     const loc = getCurrentLocation()
     loc.path[1] = setting.ids.SettingApp
-    loc.path[2] = sp.id
+    loc.path[2] = sp.name
     loc.path.length = 3
     navigate(loc)
   }
 
   function signOut (): void {
+    closePopup()
```

Comment:
Why? User will be redirected and popup will be destroyed. Or not?

Reply:
this change is not Belong to this PR any more :-)

Reply:
Fixed.